OtripleS.Web.Api/Services/Foundations/Attendances/AttendanceService.cs

```diff
@@ -29,19 +29,21 @@ public partial class AttendanceService : IAttendanceService
             this.loggingBroker = loggingBroker;
         }
 
-        public ValueTask<Attendance> ModifyAttendanceAsync(Attendance attendance) =>
-        TryCatch(async () =>
-        {
-            ValidateAttendanceOnModify(attendance);
-            Attendance maybeAttendance = await storageBroker.SelectAttendanceByIdAsync(attendance.Id);
-            ValidateStorageAttendance(maybeAttendance, attendance.Id);
+        public ValueTask<Attendance> CreateAttendanceAsync(Attendance attendance) =>
```

Comment:
Fix the alignment in all the methods
<img width="689" alt="image" src="https://user-images.githubusercontent.com/89320816/159541650-600c8b02-5ccf-4acb-a2e4-59d9db40ce8e.png">